src/app.js

```diff
@@ -197,11 +199,14 @@ export function app(app) {
     function removeChild() {
       parent.removeChild(element)
     }
+    recoverNode(node)
```

Comment:
@andyrj @zaceno What happens when you use `onremove`, and therefore must remove the element yourself using:

```js
element.parent.removeChild(element)
```

I guess we can assume (can we?) the user is going to remove the element, but is it okay to recycle the node this way?